src/containers/shared/css/global.scss

```diff
@@ -285,7 +285,18 @@ div.react-stockchart div {
       background: $nft;
     }
   }
+
+  &.XChainCreateBridge,
```

Comment:
You can use the mixin `@transaction-type-base($sidechain)`.

See https://github.com/ripple/explorer/pull/225/files#diff-cf9bdf5a51971a13a2bed1f57e95ab01982c2ef37ad6e03553275374ec945bf4R96-R101

Reply:
I haven't rebased this in a while, it's pretty outdated. I'll open it when I'm ready.